apps/pie-microsite/src/tokensMetadata.json

```diff
@@ -189,232 +189,383 @@
       "default": {
         "background-default": {
           "category": "pageBackgrounds",
-          "description": "Default page background"
+          "description": "Default page background",
+          "globalToken": "$color-mozzarella-5"
```

Comment:
Do we have a plan to tie this up with the pie-design-tokens package? Otherwise I can see this getting out of sync in future

Reply:
it's going to get massively out of sync yes, and no there are no plans currently. we're in a weird stage between some kind of integration tool such as style dictionary and this. I would like us to eventually move this file into the tokens package but there is no immediate plan to

Reply:
I think at the end of the foundations epic we should have a separate ticket to either:

- Move this file into the tokens repo
or
- Move the design tokens into this repo (and move this file into that package alongside the tokens)

Probably the second one makes the most sense potentially, as it makes sense to move the tokens into our central repo at that point.

Reply:
I like the second option

Reply:
as do I :)